Source/App/EbAppInputy4m.c

```diff
@@ -314,8 +314,18 @@ int32_t read_y4m_frame_delimiter(EbConfig_t *cfg) {
     fresult = fgets((char *)bufferY4Mheader, sizeof(bufferY4Mheader), cfg->inputFile);
 
     if (fresult == NULL) {
-        assert(feof(cfg->inputFile));
-        return EB_ErrorNone;
+        // Could be at end of file
+        // Rewind file and skip over header
+        fseek(cfg->inputFile, 0, SEEK_SET);
+        char buffer[YFM_HEADER_MAX];
+        fresult = fgets(buffer, sizeof(buffer), cfg->inputFile);
```

Comment:
Can it be just fseek by size of YFM_HEADER_MAX? You don't actually use `buffer`.

Reply:
The header is read with fgets:
char * fgets ( char * str, int num, FILE * stream );

fgets reads characters from stream and stores them as a C string into str until (num-1) characters have been read or either a newline or the end-of-file is reached, whichever happens first.

The buffer is allocated to a maximum size (YFM_HEADER_MAX = 80), but fewer than that number of bytes are actually read.  The number of bytes read varies from file to file because the width, height, and other values are stored as ascii strings separated by tokens.

Reply:
I see, thx.